package.json

```diff
@@ -32,9 +37,9 @@
   "author": "Alvaro Medina Ballester <me@alvaromb.com>",
   "license": "MIT",
   "bugs": {
-    "url": "https://github.com/APSL/react-native-keyboard-aware-scroll-view/issues"
+    "url": "https://github.com/ExodusMovement/react-native-keyboard-aware-scroll-view/issues"
   },
-  "homepage": "https://github.com/APSL/react-native-keyboard-aware-scroll-view#readme",
+  "homepage": "https://github.com/ExodusMovement/react-native-keyboard-aware-scroll-view#readme",
```

Comment:
if we want to contribute this to upstream, we can do this separately

Reply:
good now

Reply:
i still see ExodusMovement changes in package.json, but i guess it doesn't matter, we can remove them if we want to create the upstream PR